### DISCUSSION TITLE
lotus-miner run，无法正常启动。一运行，就开始等待同步，是需要同步完成了，才能正常启动 lotus-miner run 吗？

### DISCUSSION THREAD

**Discussion by k178613133** [filecoin-project/lotus#4605] [General Q&A] [answered]
lotus-miner run，无法正常启动。一运行，就开始等待同步，是需要同步完成了，才能正常启动 lotus-miner run 吗？

**Comment by CD-oursoft** [comment, marked as answer]
是的

**Comment by armatrix** [comment]
需要同步完成
现在同步好像有点问题

**Comment by k178613133** [comment]
等同步完成了，又有新的块则增加了，又得继续同步，按照目前同步块的速度 2秒1个，这样怕是永远都没法同步完成了哇。。。。。。有矿工启动起来的嘛？难受。。

**Comment by armatrix** [comment]
你可以添加 --nosync 选项来实现，默认的是false，即等待同步完成

**Comment by k178613133** [comment]
这样不等待同步 ，开始矿工，会有啥影响吗？
